src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -132,6 +133,7 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     REGISTER_PASS(manager, WeightsDequantizeToFakeQuantize)
 
     auto common_fusions = manager.register_pass<GraphRewrite>();
+    ADD_MATCHER(common_fusions, ScaledDotProductAttentionDecomposition)
```

Comment:
Can the `ScaledDotProductAttentionDecomposition` be moved from `common_fusions` to the `decompositions` (below L148)?
https://github.com/openvinotoolkit/openvino/blob/866ec162c74d3a6081463c173d39255937788a32/src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp#L148-L149
```suggestion
    ADD_MATCHER(decomp, ScaledDotProductAttentionDecomposition)
```